bioconda_utils/cran_skeleton.py

```diff
@@ -0,0 +1,136 @@
+from optparse import OptionParser
+import subprocess as sp
+import re
+from itertools import zip_longest
+import argparse
+import yaml
+
+
```

Comment:
Can we add a short description about the purpose of this file?
This is the file which you started at the ELIXIR hackathon in Paris, right? The port of conda_r_helper_script?

If so we could just add, that this file cleans-up normal cran skeletons to make it conda-forge compliant.